graalpy-eval-demo/README.md

```diff
@@ -0,0 +1,57 @@
+# GraalPy Evaluating Demo
+This small application demonstrates how to use GraalPy to evaluate a Python file from a Java application using Maven.
```

Comment:
This small application demonstrates how to use GraalPy to evaluate a Python file from a Java application, built with Maven.